core/src/test/java/com/google/cloud/sql/core/CloudSqlCoreTestingBase.java

```diff
@@ -62,6 +62,8 @@ public class CloudSqlCoreTestingBase {
 
   ListenableFuture<KeyPair> clientKeyPair;
 
+  public CloudSqlCoreTestingBase() {}
```

Comment:
FWIW In my opinion, this base class deserves to be split up into small composable units. As is, it's an awkward tool.

Reply:
+1. Maybe in a different PR.